scripts/build.py

```diff
@@ -1097,6 +1097,11 @@ def generate_build_directory():
 
 
 def build():
+    """The main method of this script.
```

Comment:
Can you add one blank line below this one?

Otherwise, looks great. Thanks!

Reply:
but sir it will giving linting isssue of "trailing white spaces "

Reply:
Hi @import-keshav, do not add "spaces" after this sentence, instead just add a line break. There will not be any such error then. Also, all the commits in this PR have been made using GitHub's online editor which bypasses all pre-submit checks. Can you please use your local editor for making and committing changes? Thanks!

Reply:
sir @apb7 while pushing i am getting this error.
I googled a lot but i was still not found a solution.
thats why i tried this.
![error_part_1](https://user-images.githubusercontent.com/34139754/47966928-884d9e00-e025-11e8-9559-090cc96b6c77.png)
![error_part_2](https://user-images.githubusercontent.com/34139754/47966931-8edc1580-e025-11e8-8903-ee2327d555af.png)